Packs/Zendesk/Integrations/Zendeskv2/README.md

```diff
@@ -788,3 +791,186 @@ To set up the mirroring:
+| Zendesk.UserGroup.url | String | The url that points to the user's API record. | 
+| Zendesk.UserGroup.user_fields | Unknown | The user fields as shown in the Zendesk user interface. | 
+| Zendesk.UserGroup.verified | Boolean | Whether any of the user's identities is verified. | 
+
+#### Command example
+```!zendesk-group-user-list group_id=12345 limit=1```
+#### Context Example
+```json
+{
+    "Zendesk": {
+        "UserGroup": [
+            {
+                "active": true,
+                "alias": "",
+                "created_at": "2022-03-27T08:42:06Z",
+                "custom_role_id": 4678497517981,
+                "default_group_id": 4678483739805,
+                "details": "",
+                "email": "test@user.com",
+                "external_id": null,
+                "iana_time_zone": "Asia/Jerusalem",
+                "id": 1908275070333,
+                "last_login_at": "2023-05-31T11:13:41Z",
+                "locale": "en-US",
+                "locale_id": 1,
+                "moderator": true,
+                "name": "Admin",
+                "notes": "",
+                "only_private_comments": false,
+                "organization_id": 4678483740317,
+                "phone": null,
+                "photo": null,
+                "report_csv": true,
+                "restricted_agent": false,
+                "role": "admin",
+                "role_type": 4,
+                "shared": false,
+                "shared_agent": false,
+                "shared_phone_number": null,
+                "signature": "",
+                "suspended": false,
+                "tags": [],
+                "ticket_restriction": null,
+                "time_zone": "Asia/Jerusalem",
+                "two_factor_auth_enabled": null,
+                "updated_at": "2023-05-31T11:13:41Z",
+                "url": "https://some-url/api/v2/users/1908275070333.json",
+                "user_fields": {},
+                "verified": true
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Zendesk Group Users:
+>|Id|Name|Email|Role|CreatedAt|
+>|---|---|---|---|---|
+>| 1908275070333 | Admin | test@user.com | admin | 2022-03-27T08:42:06Z |
+
+### zendesk-group-list
+
+***
+Get Zendesk groups. 
+Allowed for: Admins, Agents.
+
+#### Base Command
+
+`zendesk-group-list`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| limit | Maximum number of results to return. Default is 50. | Optional | 
+| page_size | The page size (used for pagination). | Optional | 
+| page_number | The page number (used for pagination). | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Zendesk.Group.created_at | Date | The time the group was created. | 
+| Zendesk.Group.default | Boolean | If the group is the default one for the account. | 
+| Zendesk.Group.deleted | Boolean | Deleted groups get marked as such. | 
```

Comment:
@ShirleyDenkberg Not sure the change is correct here. 
Basically, if the group is deleted it will return True else False. Just a boolean argument to indicate if the group was deleted or not

Reply:
So can we change it to  "Indicate if the group was deleted."